### ISSUE TITLE
How can I deal with GraphQL::Schema defaults changes?

### ISSUE THREAD

**Issue by sato-s** [rmosolgo/graphql-ruby#3409]
According to change log, we had this breaking change.

```
GraphQL::Schema defaults to GraphQL::Execution::Interpreter, GraphQL::Analysis::AST, GraphQL::Pagination::Connections, and GraphQL::Execution::Errors. (#3145) To get the previous (deprecated) behaviors:
```

I know we can keep original behavior with this workaround.

```
# Revert to deprecated execution behaviors:
use GraphQL::Execution::Execute
use GraphQL::Analysis
# Disable the new connection implementation:
self.connections = nil
```

But, how can I upgrade to new behavior?
Which classes should we take care of?

**Comment by rmosolgo** [comment]
Regarding `Execution::Interpreter` and `Analysis::AST`, you can find a writeup on those modules here: https://graphql-ruby.org/queries/interpreter.html. (More meta-documentation, in case you're curious about the design: https://rmosolgo.github.io/ruby/graphql/2019/01/29/a-new-runtime-in-graphql-ruby-1-dot-9.html). 

For new connections, they're documented here: https://graphql-ruby.org/guides#pagination-guides

The biggest _change_ for new connections is that the unfiltered list of items is now called `.items` instead of `.nodes` -- that's almost the only change required to adopt them, in most cases. (There was another compat problem fixed by #3360.) 

Generally speaking, if none of the migration notes described in the docs above sound familiar to you, and your tests all pass, then you're probably fine to upgrade without worrying about it. But for schemas with deep integration with the old runtime, migration is a bit more work.

If you try the upgrade and run into specific problems, or if you have specific questions about the features or how to migrate, please let me know!

**Comment by sato-s** [comment]
@rmosolgo 
Thanks for your reply!
I'll look into these documents!
